sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClientBuilder.java

```diff
@@ -267,6 +282,11 @@ public HttpClient build() {
             }
         }
 
+        // Do not follow redirects by default. We should let the pipeline have more control of this.
+        if (!this.followRedirects) {
+            httpClientBuilder.followRedirects(false);
+        }
```

Comment:
Couldn't this just be

```suggestion
        httpClientBuilder.followRedirects(followRedirects);
```

since `boolean` is false by default